services/src/main/java/org/keycloak/services/cors/DefaultCors.java

```diff
@@ -117,7 +117,10 @@ public Cors allowedMethods(String... allowedMethods) {
 
     @Override
     public Cors exposedHeaders(String... exposedHeaders) {
-        this.exposedHeaders = new HashSet<>(Arrays.asList(exposedHeaders));
+        if (this.exposedHeaders == null) {
+            this.exposedHeaders = new HashSet<>();
+        }
+        this.exposedHeaders.addAll(Arrays.asList(exposedHeaders));
```

Comment:
This changes the behaviour a bit to add `exposedHeaders` to `this.exposedHeaders` instead of overriding it how it was before. This is also not compatible with other methods like `allowedMethods` etc.

Can we perhaps either:
- Add new method like `addExposedHeaders` to both `Cores` and `DefaultCors`
- Rename the existing method to something like `addExposedHeaders` .

I prefer the first option, but second is fine if you prefer it for some reason

Reply:
Option 1 done! Let's see the tests!